server/data_updates/00009_20181114-090110_vocabularies.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Creation: 2018-11-14 10:31
+
+from superdesk.commands.data_updates import DataUpdate
+from superdesk import get_resource_service
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'vocabularies'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        vocabularies_service = get_resource_service('vocabularies')
+        for vocabulary in vocabularies_service.get(req=None, lookup=None):
+            if vocabulary.get('single_value', False):
+                value = 'single selection'
+            else:
+                value = 'multi selection'
+            vocabularies_service.patch(vocabulary['_id'], {'selection_type': value})
```

Comment:
should not the `single_value` key be removed?

Reply:
We only can set null for it.

Reply:
@ioanpocol with mongo we can remove it.

Reply:
done